### ISSUE TITLE
-lprofiler: -ltcmalloc: No such file or directory

### ISSUE THREAD

**Issue by jiapei100** [colmap/colmap#2049]
**Describe the bug**
Not able to build out **colmap** with the following 2 error messages:

```console
......
/usr/bin/ld: cannot find -lprofiler: No such file or directory
/usr/bin/ld: cannot find -ltcmalloc: No such file or directory
.....
```

**To Reproduce**
Just cmake build ...

**Expected behavior**
Successfully built.

**Screenshots**
N/A

**Environment:**
 - OS: [e.g. Windows 11] Ubuntu 22.04
 - COLMAP Version [e.g. 3.8 or git commit hash] : 3.8
 - Capture Device [e.g. iPhone X]: N/A

**Comment by jiapei100** [comment]
Problem solved by remove the following 2 lines in **CMakeLists.txt**.

```txt
......
#    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -lprofiler -ltcmalloc")
#    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -lprofiler -ltcmalloc")
......
```

**Comment by ahojnnes** [comment]
It looks like you accidentally enabled profiling but do not have appropriate dependencies on your system installed.

**Comment by jiapei100** [comment]
@ahojnnes 

Yes, I enabled `profiling` ... But, what are the appropriate dependencies?

**Comment by ahojnnes** [comment]
Closing: user self-resolved by removing profiling flags.
